services/metadata/frontend/routes.js

```diff
@@ -0,0 +1,7 @@
+import Content from './content/Content.svelte';
+import ProjectPage from './content/ProjectPage.svelte';
+
+export default {
+  '/': Content,
+  '/project/:id': ProjectPage
+}
```

Comment:
is there a reason for this to be JavaScript rather than TypeScript?

Reply:
Good point. I'm not sure if TS is fully supported, will give it a try. Was just following the guide, where JS is used for the routes.

Reply:
if it works in TS, that would be more consistent. But having this in JS doesn't hurt either.